### PR TITLE
Change preloader for Polygon to include RPC notice

### PR DESCRIPTION
**Pull request by mannyreimi88**

### CODE REVIEW

File: features/Jars/JarList.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { useJarData } from "./useJarData";
 import { Connection } from "../../containers/Connection";
 import { JAR_ACTIVE, JAR_YEARN } from "../../containers/Jars/jars";
 import { backgroundColor, pickleGreen } from "../../util/constants";
-import { NETWORK_NAMES } from "containers/config";
+import { NETWORK_NAMES } from "../../containers/config";
```
Comment (larrythecucumber321) [inline features/Jars/JarList.tsx:13]: This isn't necessary, as I've enabled absolute path imports

File: features/Gauges/GaugeList.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ import {
 import { JarApy } from "../../containers/Jars/useJarsWithAPY";
 import { useUniPairDayData } from "../../containers/Jars/useUniPairDayData";
 import { Jars } from "../../containers/Jars";
+import { NETWORK_NAMES } from "../../containers/config";
```
Comment (larrythecucumber321) [inline features/Gauges/GaugeList.tsx:23]: You can undo all the changes in this file. Pickle on Polygon will never call this file. Ditto for `FarmList`

File: features/Jars/JarList.tsx (hunk 1 above)
Comment (mannyreimi88) [reply features/Jars/JarList.tsx:13]:
```suggestion
import { NETWORK_NAMES } from "containers/config";
```

File: features/Gauges/GaugeList.tsx (hunk 2 above)
Comment (mannyreimi88) [inline features/Gauges/GaugeList.tsx:23]:
```suggestion

```

File: features/Gauges/GaugeList.tsx [hunk 3]
```diff
@@ -49,7 +50,7 @@ const GreenSwitch = withStyles({
 })(Switch);
 
 export const GaugeList: FC = () => {
-  const { signer } = Connection.useContainer();
+  const { signer, chainName } = Connection.useContainer();
```
Comment (mannyreimi88) [inline features/Gauges/GaugeList.tsx:53]:
```suggestion
  const { signer } = Connection.useContainer();
```

File: features/Gauges/GaugeList.tsx [hunk 4]
```diff
@@ -67,8 +68,10 @@ export const GaugeList: FC = () => {
     return <h2>Please connect wallet to continue</h2>;
   }
 
-  if (!gaugeData) {
+  if (!gaugeData && chainName !== NETWORK_NAMES.POLY) {
```
Comment (mannyreimi88) [inline features/Gauges/GaugeList.tsx:71]:
```suggestion
  if (!gaugeData) {
```

File: features/Gauges/GaugeList.tsx [hunk 5]
```diff
@@ -67,8 +68,10 @@ export const GaugeList: FC = () => {
...
     return <h2>Loading...</h2>;
+  } else if (!gaugeData && chainName === NETWORK_NAMES.POLY) {
+    return <h2>Loading...(if you have been waiting more than a few seconds, you may be rate-limited, consider changing to a different Polygon RPC such as 'https://rpc-mainnet.matic.network' or 'https://rpc-mainnet.maticvigil.com')</h2>;
```
Comment (mannyreimi88) [inline features/Gauges/GaugeList.tsx:74]:
```suggestion

```

**Pull request merged**